docs/learn-legitimate-projects.md

```diff
@@ -1,242 +1,162 @@
 
-## Signs of a legitimate project
+The same thing goes for projects that have the "Polka" prefix in their name. Many projects use that to associate themselves with the ecosystem, some legitimately and others only to piggyback on Polkadot's reputation.
 
-#### 1. It has received a Web3 Grant
+## "Hard" metrics to look for when you DYOR
 
-[Web3 Foundation Grants](https://github.com/w3f/Open-Grants-Program) are a way to subsidise projects
-that are building on Polkadot. They have very specific criteria in order to apply and are thorougly
-reviewed by the review team. So, having their application approved is a first indication that the
-team is committed to their project. Once they have delivered their first milestone, they can add the
-["Web3 Grants" badge](https://github.com/w3f/General-Grants-Program/blob/master/grants/grant-badge-guidelines.md)
-on their project's site.
+#### 1. Association with entities you trust
 
-Having received a grant is a strong indicator that a project is legitimate. And if they have
-delivered all of their milestones, then it's most likely of reasonable quality. However, a grant is
-not a guarantee on the continuity of the project and it can be terminated at any point if they
-violate the rules of the program, so that's something to look out for.
+New projects usually try to increase their credibility by associating themselves with well-known entities. The thinking is simple: "These entities that have a good reputation trust us, so if you trust them, by association, you should trust us too". And that's a valid argument and a strong indicator of the legitimacy of a project.
 
-Finally, a grant **is not** an endorsement by or a partnership with Web3 Foundation.
+For example, if a project has received a [Web3 Foundation Grant](https://github.com/w3f/Open-Grants-Program), this is an indication that the project is indeed building on the Polkadot ecosystem, and if they have delivered all of their milestones, then their code is most likely of reasonable quality. 
+
+And Web3 Foundation is not the only entity in the ecosystem that gives grants. There are other reputable teams in the ecosystem that have developed platforms or prospective parachains and give grants for projects to build on or expand their project. These are also indicators that a project is committed to building on the broader Polkadot ecosystem. 
+
+Or receiving funding from VCs that you trust and are known to be involved with other reputable Polkadot projects can also be a good indicator. Or participating in the [Substrate Builders Program](https://www.substrate.io/builders-program/).
```

Comment:
Take out the "Or's"

Reply:
Same here. It's a continuation of the examples we're giving. "This, for example, can be a good point. And this one. Or this one. Or this fourth one"